tasks/simplifications.json

```diff
@@ -191,6 +191,14 @@
 		"skipfailures": "no"
 	},
 
+	"ca.statcan.geo.importgeography_da": {
+		"simplification": "mapshaper",
+		"geomfield": "wkb_geometry",
+		"factor": "7",
```

Comment:
I have tested it and ` "factor": "7"` can be a good approach for this geometry. The rest of the simplification parameters are OK